### ISSUE TITLE
Exempt VAMC System VA Police content type from the outdated content report and trigger

### ISSUE THREAD

**Issue by xiongjaneg** [department-of-veterans-affairs/va.gov-cms#16120]
## User Story or Problem Statement
As a VAMC editor, I don't want to get outdated content reports for the VAMC System VA Police page that I won't be able to edit during the MVP stage.

For Police MVP, the VAMC System VA Police page won't have any fields that are editable by local editors. Therefore, we don't want outdated content reporting to include this page and alert editors since they can't do anything to it during MVP.

## Acceptance Criteria
- [ ] Testable_Outcome_X
- [ ] Testable_Outcome_Y
- [ ] Testable_Outcome_Z
- [ ] Requires design review
- [ ] Requires accessibility review

**Comment by swirtSJW** [comment]
@xiongjaneg I don't think we should frame this toward the MVP.  It would be a year or more before this page would ever show on anything related to outdated content.  We should look toward a bit farther out in our decision.
